plugins/search/providers/elastic_search/lib/kElasticSearchManager.php

```diff
@@ -194,7 +200,7 @@ private function saveToSphinxLog($object, $params, $shouldSyncElastic)
 
     private function shouldSyncElastic($object)
     {
-        $key = kCurrentContext::$ks_partner_id . "_" . kCurrentContext::$service . "_" . kCurrentContext::$action . "_" . $object->getElasticIndexName();
+        $key = kCurrentContext::$ks_partner_id . "_" . kCurrentContext::$service . "_" . kCurrentContext::$action . "_" . kBaseESearch::getElasticIndexNamePerPartner($object->getElasticIndexName(),$object->getPartnerId());
```

Comment:
the key takes partner id from kCurrentContext::$ks_partner_id but the function use $object->getPartnerId() are they the same?